### PR TITLE
[NEW] Print transcript on mobile condition

### PR DESCRIPTION
**Pull request by AlexanderKanakis**
For issue: https://github.com/WideChat/Rocket.Chat/issues/1200

Related PRs: https://github.com/WideChat/Rocket.Chat/pull/1201

### CODE REVIEW

File: src/components/Messages/Message/index.js [hunk 1]
```diff
@@ -43,6 +44,11 @@ const onClickTranscript = async () => {
 	await handleTranscript();
 };
 
+const canPrintTranscript = () => {
+	const { transcript, printMobileTranscript } = store.state.config.settings;
+	return transcript && isMobile() ? printMobileTranscript : true;
+};
+
```
Comment (Shailesh351) [inline src/components/Messages/Message/index.js]: @AlexanderKanakis When the transcript functionality is disabled (Not mobile transcript) It returns true. So it shows the `Save Transcript` button.

You can try disabling both transcript settings.

Also please check the comment I have added in the server repo for naming confusion. I think it applies here as well

**Comment by ear-dev** [comment]
@Shailesh351 good to go?

File: src/components/Messages/Message/index.js [hunk 2]
```diff
@@ -43,6 +44,11 @@ const onClickTranscript = async () => {
 	await handleTranscript();
 };
 
+const canPrintTranscript = () => {
+	const { transcript, enableTranscriptMobile } = store.state.config.settings;
+	return transcript && isMobile() ? enableTranscriptMobile : true;
+};
+
```
Comment (Shailesh351) [inline src/components/Messages/Message/index.js]: @AlexanderKanakis When the transcript functionality is disabled (Not mobile transcript) It returns true. So it shows the `Save Transcript` button.

You can try disabling both transcript settings and it still shows the `Save Transcript` button

**Review by Shailesh351 — APPROVED** [review]

File: src/components/Messages/Message/index.js (hunk 2 above)
Comment (ear-dev) [reply src/components/Messages/Message/index.js]: good catch @Shailesh351 ........  thanks.

**Pull request merged**